StarRating/themes/StarRating_template.less

```diff
@@ -3,8 +3,9 @@
 .d-star-rating{
 	/* Do not modify the display style */
 	display: inline-block;
-	font-size: @star-icon-size;
+	font-size: 285%;
```

Comment:
don't we want star rating font size to be set independently of the overall font ? I mean a similar application, with same font size might still want very big star rating for some reasons?